.gitignore

```diff
@@ -66,6 +66,9 @@ refine-dev.ini
 # Ignore Node modules that might inadvertently install at current path instead of user or configured env var NODE_PATH 
 node_modules/
 
+# Ignore locally build snaps
+*.snap
+
```

Comment:
We can probably also remove this change since we are not building the Snaps in this repository